src/main/java/dev/unnm3d/rediseconomy/currency/Currency.java

```diff
@@ -663,14 +650,7 @@ private void setPlayerMaxBalanceLocal(UUID uuid, double amount) {
     }
 
     private void setPlayerMaxBalanceCloud(UUID uuid, double amount) {
```

Comment:
Wherever this method is called, use economyStorage.updatePlayerMaxBalance instead